src/ui/pages/CredentialDetails/CredentialDetails.tsx

```diff
@@ -121,6 +122,12 @@ const CredentialDetails = () => {
       handleSetFavourite(params.id);
     }
     dispatch(setCredsCache(creds));
+
+    const newArchivedCredsCache = await Agent.agent.credentials.getCredentials(
```

Comment:
I don't think we need the changes for this file since the useEffect in Credentials.tsx will auto fetch the archived credentials after an action happens (which triggers the toast message).

I don't particularly like relying on the toast messages but was hoping this PR would be smaller.

If the useEffect doesn't cover these scenarios - maybe we can just manually calculate the new archived creds list and update Redux instead of calling `getCredentials` again.